src/token.ts

```diff
@@ -120,6 +119,20 @@ export const STAR: Token = {token: 'op', str: '*'};
 export const IMMEDIATE: Token = {token: 'op', str: '#'};
 export const ASSIGN: Token = {token: 'op', str: '='};
 
+// CS -> CS token alias map
+export const CS_TOKEN_ALIAS_MAP = new Map<string, string>([
```

Comment:
(nit) You can omit the explicit template params here, TS will infer `<string, string>`

Reply:
Yeah, as you deduced, I learned TypeScript yesterday :P